metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -488,9 +488,12 @@ class Compilers(
                         |}""".stripMargin
     val modifiedInput =
       originInput.copy(value = withOuter)
-    val adjustLspData = AdjustLspData.create(pos => {
-      new LspPosition(pos.getLine() - 1, pos.getCharacter() - ident.size)
-    })
+    val adjustLspData = AdjustLspData.create(
+      pos => {
+        new LspPosition(pos.getLine() - 1, pos.getCharacter() - ident.size)
+      },
+      filterOutLocations = { loc => loc.getUri().isWorksheet }
```

Comment:
we shouldn't adjust positions outside of the worksheets.

Reply:
🤔 not work for now, but I'm wondering for clarity in the future if we should try to unify where we have adjustments done. For example we check here for worksheets and sbt and adjust accordingly. However, Ammonite also has some adjustments that we do, but those live in 	`Ammonite.scala`. I can see how that could be confusing from someone on the outside. Or, is it a good distinction to have that adjustments related to the PC are in `AdjustLspData` and other things live in their corresponding places? What do you think?

Reply:
Yeah, I will move the adjustments to separate locations